src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -772,6 +772,56 @@
         "6.0.0.0": "6.0.0"
       }
     },
+    "Microsoft.Extensions.Hosting.Systemd": {
```

Comment:
Where do things like

```xml
<PackageTags>hosting</PackageTags>
```

go in our build infrastructure? Both packages included that tag before.